content/rc/rc-quickstart.md

```diff
@@ -50,23 +61,23 @@ If you would rather customize your subscription and database, select **Create a
 
     {{<image filename="images/rc/quickstart-database-overview.png" width="75%" alt="Configuration tab showing details of your new database." >}}{{< /image >}}
 
+1.  Confirm your database is active  
```

Comment:
```suggestion
1.  
```
I agree that there needs to be a number here, but this step doesn't need a heading and could just start with the next line.